<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="feiwin7vm67" ignored="0" name="SalesOrder" tests="2" failures="1" timestamp="19 Sep 2018 14:15:44 GMT" time="7.729" errors="0">
  <testcase name="MTC001_VerifyResponseForValid_MstrCustID_accept_JSON" time="6.23" classname="TestRestAPI.SalesOrder.listTransactionSummaryforSourceBid"/>
  <testcase name="MTC002_VerifyResponseForValid_MstrCustID_accept_XML" time="0.375" classname="TestRestAPI.SalesOrder.listTransactionSummaryforSourceBid">
    <failure type="java.lang.AssertionError" message="expected [12345] but found [143312]">
      <![CDATA[java.lang.AssertionError: expected [12345] but found [143312]
	at TestRestAPI.SalesOrder.listTransactionSummaryforSourceBid.MTC002_VerifyResponseForValid_MstrCustID_accept_XML(listTransactionSummaryforSourceBid.java:333)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- MTC002_VerifyResponseForValid_MstrCustID_accept_XML -->
</testsuite> <!-- SalesOrder -->
